packages/flutter_tools/lib/src/base/process.dart

```diff
@@ -10,6 +10,8 @@ import '../globals.dart';
 
 typedef String StringConverter(String string);
 
+final Map<String, String> childEnvironmentOverrides = new Map<String, String>();
```

Comment:
I'm worried that this won't act as overrides but will entirely replace the env vars. Perhaps init it to a copy of the current env. vars?